### PR TITLE
Crew create af

### PR DESCRIPTION
**Pull request by AutumnFoto**
## Description
crew create

## Related Issue
crew create

## How Can This Be Tested?
git fetch crewCreateAF
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/components/forms/addCrewForm.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            <button id="add-crew-btn" type="submit" class="btn btn-info"><i class="far fa-calendar-plus"></i> Add  Crew</button>
+          </form>
+          `
+  );
+  $('#add-crew-btn').on('click', (e) => {
+    e.preventDefault();
+    const data = {
+      name: $('#name').val() || false,
+      role: $('#role').val() || false,
+      uid: $('#uid').val() || false,
```
Comment (chris-calhoun) [inline src/javascripts/components/forms/addCrewForm.js:26]: Don't need this line anymore

File: src/javascripts/components/forms/addCrewForm.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            '<div class="alert alert-success" role="alert">Your Crew Was Added!</div>'
+          );
+        })
+        .catch((error) => console.warn(error));
+      setTimeout(() => {
+        $('#sucess-message').html('');
+      }, 3000);
+      $('#name').val('');
+      $('#role').val('');
+      $('#uid').val('');
```
Comment (chris-calhoun) [inline src/javascripts/components/forms/addCrewForm.js:47]: or this one.

**Review by wmmcinturff — APPROVED** [review]
Approve removal of uid from form and card

**Review by chris-calhoun — APPROVED** [review]
👍

**Pull request merged**